### DISCUSSION TITLE
Hatch with buttons gap became very large since the last update?

### DISCUSSION THREAD

**Discussion by ubertpendragon** [FakeFishGames/Barotrauma#14021] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Hatch With Buttons gap had a growth spurt since the last update:

Before

![image](https://github.com/FakeFishGames/Barotrauma/assets/73229309/273f1c0b-18b6-4adb-9af8-492db584d58e)

After

![image](https://github.com/FakeFishGames/Barotrauma/assets/73229309/76a2bff6-d15d-469e-b145-91414f2b6aa7)

The green rectangle also seems misaligned compared to doors with buttons, so I'm not sure what happened

### Version

v1.4.6.0 (Blood in the Water Update, hotfix 2)

**Comment by JeroonBosch** [comment]
Oops. Issue has been spotted and fix has been pushed to the private repo. Thanks for reporting.

**Reply by JeroonBosch** [reply]
A solution has been merged
